.travis.yml

```diff
@@ -1,21 +1,14 @@
 language: go
-
-env:
-  global:
-    - DEPS=$HOME/deps
-    - PATH=$DEPS/bin:$PATH
-    - PATCH="v8.0.0134"
-
-install: |
-  git config --global user.email "you@example.com"
-  git config --global user.name "Your Name"
-
-  # check out if we can pre-compiled Vim releases somehow,
-  git clone --branch $PATCH --depth 1 https://github.com/vim/vim
-  cd vim
-  ./configure --prefix=$DEPS --with-features=huge --disable-gui
-  make
-  make install
-  cd -
-
-script: ./scripts/test.sh
+cache:
+  directories:
+    - /tmp/vim-go-test
```

Comment:
How will this affect the builds when we decide to update to a newer patch version of the installed versions?

Reply:
The cache is per-pr, so we should be fine.

Reply:
Travis docs say the cache for a branch/pr may come from master, too. Couldn't that be a problem for us?

Reply:
I don't know, but it's never has been for me in the past. But if it is, we can manually delete it in the Travis UI. Updating patch-levels is not going to be a super-frequent event I think.

Reply:
Could also add patchlevels to the paths in `/tmp/vim-go`. I think there was a reason I didn't, but I forgot what it was (been a few weeks).

Reply:
We shouldn't have to go change Travis configuration because of a change in which Vim versions we support.

I understand the draw of caching this, but long term I think it will cause us to spend cycles debugging what we could solve now by not caching (though at the cost of longer build times).

I'll digress, but please consider (maybe even test to try to invalidate each of our understandings) the concerns I raised.

Reply:
I don't think it'll be an issue, but OTOH builds are already very fast so removing the cache isn't going to hurt us very much (so I've just removed it).